src/witan/gias.clj

```diff
@@ -0,0 +1,731 @@
+         "LSOA (name)"                      :lsoa-name
+         "InspectorateName (name)"          :inspectorate-name-name
+         "SENStat"                          :sen-stat
+         "SENNoStat"                        :sen-no-stat
+         "BoardingEstablishment (name)"     :boarding-establishment-name
+         "PropsName"                        :props-name
+         "PreviousLA (code)"                :previous-la-code
+         "PreviousLA (name)"                :previous-la-name
+         "PreviousEstablishmentNumber"      :previous-establishment-number
+         "OfstedRating (name)"              :ofsted-rating-name
+         "RSCRegion (name)"                 :rsc-region-name
+         "Country (name)"                   :country-name
+         "UPRN"                             :uprn
+         "SiteName"                         :site-name
+         "QABName (code)"                   :qab-name-code
+         "QABName (name)"                   :qab-name-name
+         "EstablishmentAccredited (code)"   :establishment-accredited-code
+         "EstablishmentAccredited (name)"   :establishment-accredited-name
+         "QABReport"                        :qab-report
+         "CHNumber"                         :ch-number
+         "MSOA (code)"                      :msoa-code
+         "LSOA (code)"                      :lsoa-code
+         "FSM"                              :fsm
+         "AccreditationExpiryDate"          :accreditation-expiry-date}))
+
+(comment
+  ;; Keyword column names created from CSV column lables using:
+  (let [csv-col-name->keyword-col-name (fn [s] (-> s
+                                                   (clojure.string/replace #"(?<!^)([A-Z])(?=[a-z])" "-$1")
+                                                   (clojure.string/replace #"(?<=[a-z])([A-Z])" "-$1")
+                                                   (clojure.string/replace #"[()]" "")
+                                                   (clojure.string/replace #" +" "-")
+                                                   (clojure.string/lower-case)
+                                                   keyword))]
+    (into (sorted-map-by (partial compare-mapped-keys csv-col-name->order)) 
+          (map (fn [s] [s (csv-col-name->keyword-col-name s)]))
+          csv-col-names))
+
+  )
+
+(def col-name->order
+  (let [m (update-keys csv-col-name->order csv-col-name->col-name)]
+    (into (sorted-map-by (partial compare-mapped-keys m)) m)))
+
+(def col-names
+  (apply sorted-set-by
+         (partial compare-mapped-keys col-name->order)
+         (keys col-name->order)))
+
+(def col-name->csv-col-name
+  (into (sorted-map-by (partial compare-mapped-keys col-name->order)) 
+        (set/map-invert csv-col-name->col-name)))
+
+(def col-name->label
+  "Map column name to label."
+  (into (sorted-map-by (partial compare-mapped-keys col-name->order))
+        (update-keys csv-col-name->label csv-col-name->col-name)))
+
+
```

Comment:
Do we need something like this to do the namespaced keywords?

```
  (defn fqkn->csv-header [k]
    (str (namespace k) "/" (name k)))
  (fqkn->csv-header ::clerk/width)
  "nextjournal.clerk/width"
  (keyword "nextjournal.clerk/width")
  :nextjournal.clerk/width
  (= ::clerk/width (keyword "nextjournal.clerk/width"))
  true
```

Reply:
Yes, with some extension to cope with non-namespaced keyword and non-keyword column names. However it would require remembering to apply `fqkn` to the column names of any dataset with namespaced keyword names before calling `tc/write!`. Given these overheads/risks propose to not to namespace the column names.